ui/lib/core/addon/components/text-file.hbs

```diff
@@ -32,14 +32,13 @@
 <div class="field text-file box is-fullwidth is-marginless is-shadowless is-paddingless" data-test-component="text-file">
   {{#if this.showTextArea}}
     <div class="control has-icon-right">
-      <textarea
+      <Textarea
         id="input-{{this.elementId}}"
+        @value={{this.content}}
         class="textarea {{if (and (not this.showValue) this.content) 'masked-font'}}"
-        {{on "input" this.handleTextInput}}
+        {{on "change" this.handleTextInput}}
```

Comment:
@hellobontempo I changed this to an on `change` event since it's passed in as an `onChange` from the parent. I was curious why it was on `input` instead?

Reply:
It looks like the [Ember docs](https://guides.emberjs.com/release/components/built-in-components/#toc_actions) suggest to use the `input` event.

Reply:
I think the difference is that the oninput event is triggered immediately while onchange is triggered after the element loses focus.

Reply:
I missed that on the docs! Thanks for pointing that out - will fix! 🙃